services/WifiConfigFile.py

```diff
@@ -0,0 +1,70 @@
+
+import os
+import logging
+from Pab import *
+
+logger = logging.getLogger(__name__)
+
+class WifiConfigFile(object):
+
+    @staticmethod
+    def filename():
+        return "WIFICONFIG"
+
+    def __init__(self, tipi_io):
+        self.tipi_io = tipi_io
+        self.tmp = "/tmp/wificonfig"
+        self.currentRecord = 0
+
+    def handle(self, pab, devname):
+        logPab(pab)
+        op = opcode(pab)
+        if op == OPEN:
+            self.open(pab, devname)
+        elif op == CLOSE:
+            self.close(pab, devname)
+        elif op == READ:
+            self.read(pab, devname)
+        elif op == WRITE:
+            self.write(pab, devname)
+        else:
+            self.tipi_io.send([EOPATTR])
+
+    def close(self, pab, devname):
+        with open("/tmp/wificonfig",'w') as fh:
+            for line in self.records:
+                fh.write(line)
+                fh.write("\n")
+        self.tipi_io.send([SUCCESS])
+
+    def open(self, pab, devname):
+        if dataType(pab) == DISPLAY:
+            if recordLength(pab) == 0 or recordLength(pab) == 80:
+                self.currentRecord = 0
+                self.records = []
+                self.tipi_io.send([SUCCESS])
+                self.tipi_io.send([80])
+                return
+        self.tipi_io.send([EOPATTR])
+
+    def read(self, pab, devname):
+        if dataType(pab) == DISPLAY:
+            self.tipi_io.send([EEOF])
```

Comment:
This is intentionally read-only, or as I implemented, 'always empty on open'  The the output goes to /tmp/wificonfig which is just a message to TipiSuper service, and is immediately deleted.  Showing more network config details will be the job of PI.STATUS